clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/ServiceInstances.java

```diff
@@ -282,11 +279,11 @@ public void createUserProvidedServiceInstance(String newUserProvidedServiceInsta
     servicePolling.accept(guid, type);
   }
 
-  // package-level for testing visibility
+  @VisibleForTesting
   void pollServiceInstanceStatus(String serviceInstanceName, String guid, LastOperation.Type type, Duration timeout) {
     RetryConfig retryConfig = RetryConfig.custom()
       .waitDuration(pollingInterval)
-      .maxAttempts((int) (Math.max(timeout.toMillis(), defaultPollingTimeout.toMillis()) / pollingInterval.toMillis()))
+      .maxAttempts((int) (Math.max(timeout.toMillis(), pollingTimeout.toMillis()) / pollingInterval.toMillis()))
```

Comment:
Is there only one call site using `pollingTimeout` as the `timeout` argument, making this `max` comparison extraneous? Is it called elsewhere?

Reply:
👍 Good catch

Reply:
Maybe I'm misreading, but it seems to me that now we've got:
`.maxAttempts((int) (timeout.toMillis() / pollingInterval.toMillis()))` which should always evaluate to 1 and `.waitDuration(pollingInterval)` which will always wait 14 days?

Reply:
I believe that we discussed this offline:  we need to make these calls to CAPI asynchronous so we need to poll.  We decided to use Spinnaker's built-in timeout to cancel the task when its timeout is reached.  The reason it is not inlined is for testing purposes only.